portfolio/src/main/webapp/comments.js

```diff
@@ -13,24 +13,120 @@
 // limitations under the License.
 
 /**
- * Fetches comments from the server and adds it to the DOM.
+ * An object to manage the query string of a URL
+ * @const {URLSearchParams}
  */
-function getComments() {
-  fetch('/comments').then(response => response.json()).then((comments) => {
-    const commentsContainer = document.getElementById('comments-container');
+const GET_PARAMS = new URLSearchParams ({
+  imageId: '',
+  quantity: '5',
+  dateOrder: 'true'
```

Comment:
Mark had a similar comment,  but maybe change this to "dateAscending" or "dateDescending" to indicate the direction the dates are ordered in.

Reply:
Yep, yep, thanks for the names' suggestions, Aki!